deploy/cluster_role.yaml

```diff
@@ -0,0 +1,24 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: host-operator
+rules:
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
```

Comment:
this is not cluster scoped resource
![Screenshot from 2019-08-26 18-50-08](https://user-images.githubusercontent.com/7941617/63693683-5ca8d800-c832-11e9-9923-bc5dadfa6bf9.png)

Reply:
never mind, we need to keep this in clusterrole for prometheus service

Reply:
ah, sure. let me fix that 18ccf04